### DISCUSSION TITLE
[Feature]: introduce the difference and comparison between the  library and other libraries, such as cronet

### DISCUSSION THREAD

**Discussion by Haoxiqiang** [alibaba/xquic#26] [General]
### Before request a new feature

- [X] I have searched the [issues](https://github.com/alibaba/xquic/issues) of this repository and believe that this is not a duplicate.

### Describe the feature you'd like supported

Can you introduce the difference and comparison between the implementation of this library and other libraries, such as cronet?

**Comment by Kulsk** [comment]
maybe this article might be helpful:
https://www.infoq.cn/article/Ys2t7jLFRNFG9XDkeS27
